typescript/packages/lattice-client-react/src/context/lattice-config-provider.tsx

```diff
@@ -0,0 +1,52 @@
+import {type LatticeClient, type LatticeClientOptions} from '@wasmcloud/lattice-client-core';
+import * as React from 'react';
+import {useLatticeClient} from '@/context/use-lattice-client';
+
+type LatticeClientConfigOutput = typeof LatticeClient.prototype.instance.config;
+type LatticeClientConfigInput = Partial<LatticeClientOptions['config']>;
+
+type SetConfigFunction = (value: LatticeClientConfigInput) => void;
+
+export type LatticeConfigContextType = {
+  config: LatticeClientConfigOutput;
+  setConfig: SetConfigFunction;
+};
+
+export const LatticeConfigContext = React.createContext<LatticeConfigContextType>({
+  config: {} as LatticeClientConfigOutput,
```

Comment:
As far as I remember this warning will break some CI runs. Would you mind to move the Context into a different file for the HMR? Like [this](https://github.com/wasmCloud/wasmCloud/commit/314d895548107011734b2b6c936f021f255b3b66)?

Reply:
Okay, fix it.

and I'm not sure if you prefer one commit or separate ones, so I've submitted each change separately for now.